### PR TITLE
Adding new resource for mysql flexibleserver and a new filter

### PR DESCRIPTION
**Pull request by femio27**

### CODE REVIEW

File: tools/c7n_azure/c7n_azure/entry.py [hunk 1]
```diff
@@ -10,3 +10,5 @@ def initialize_azure():
     import c7n_azure.policy
     import c7n_azure.container_host.modes
     import c7n_azure.output # noqa
+
+    import c7n_azure.resources.mysql_flexibleserver
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/entry.py:14]: entry imports aren't typically for resources, their more to get modes imported for schema, resources are handled out of the resource_map. ie. this line can be removed.

Comment (femio27) [reply tools/c7n_azure/c7n_azure/entry.py]: Yes I have removed the line.

File: tools/c7n_azure/c7n_azure/resources/mysql_flexibleserver.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            resource: azure.mysql-flexibleservers
+            filters:
+              - type: configuration-parameter
+                name: tls_version
+                key: value
+                op: eq
+                value: 'TLSv1.2'
+
+    """
+
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/resources/mysql_flexibleserver.py:67]: you'll need to add a line here `schema_alias = False` as its conflicting with another filter by the same name atm.

Comment (fhemie1) [reply tools/c7n_azure/c7n_azure/resources/mysql_flexibleserver.py:68]: I have updated the script. Thank you

Comment (femio27) [reply tools/c7n_azure/c7n_azure/resources/mysql_flexibleserver.py:67]: I renamed the filter so no conflict. Thank you

**Review by kapilt — APPROVED** [review]
lgtm

**Pull request merged**